### ISSUE TITLE
Z score charts

### ISSUE THREAD

**Issue by paedza** [p2/growth-charts#25]
Excellent app 
Please could we have the Zscore charts loaded on. 
We use the who z scores from the same data set and it's an immeasurable tool to have in daily clinical practice

**Comment by paedza** [comment]
Hi. I'm new to this site. What is an enhancement label or filter.? 
Did anyone review my request? Please advise

**Comment by p2** [comment]
Yes, thanks for the feedback, I've seen it. Adding more charts is an enhancement of the app, hence the tag. ;)

**Comment by paedza** [comment]
Thank you for your reply. 

The app is truely amazing. Cuts out a lot of time! Just that we have to be cautious with final interpretations as our management of abnormal measurements depend on the z scores. Used another app .... Not exactly the best or user friendly. 

Does the enhancement mean that the additional charts will be added on? How will we know when it will be added ? 

Just a note. Auto age calculator would also be good... Minimizes user miscalculation... 

Thx again for this super app.. 

Regards 
Voshika

**Comment by p2** [comment]
Glad you like it! There are plans to add more charts, but this project right now is entirely voluntary, so it's impossible to say when I'll have time to add more charts.

**Comment by paedza** [comment]
Good morning 

 

I do hope this finds you well

 

Just doing follow-up on my previous request. 

 

The Growth chart app is great and easy to use. 

 

Is there any way we could have the Z score charts up as well? The WHO classification systems allow the use of both Z score and Percentile charts but when classifying malnutrition the Z score charts are the charts used, hence my request. We see a fair amount of malnourished children and the z score charts on the app will make a huge difference to us. 

 

Any assistance with my request would be greatly appreciated

 

Kind Regards

Voshika
